src/main/java/ca/corefacility/bioinformatics/irida/ria/web/files/ReferenceFileAjaxController.java

```diff
@@ -0,0 +1,95 @@
+package ca.corefacility.bioinformatics.irida.ria.web.files;
+
+import java.io.IOException;
+import java.util.List;
```

Comment:
This file should be in the ajax package right?

Reply:
Moved in 219a5fbdd6a0ee0e31fd61c6c443e0020191c82d